docs/getting-started/installing/digitalocean.md

```diff
@@ -10,6 +10,14 @@ title: DigitalOcean
 - [Installing Kontena Nodes](digitalocean.md#installing-kontena-nodes)
 - [DigitalOcean Plugin Command Reference](digitalocean.md#digitalocean-plugin-command-reference)
 
+## Caveats
+
+### Cannot update `cloud-config.yml` after node creation
```

Comment:
The title and phrasing here is a bit odd in terms of the remainder of the documentation, because the cloud-config is mostly just an internal detail of how the DigitalOcean plugin is implemented, so the relationship to the plugin parameters documented here is a bit vauge and unactionable. Instead, it should probably mention the specific `kontena digitalocean node create` parameters that are problematic, and how to reconfigure those later after provisioning... Those parameters and some related GitHub issues would be:

* `kontena digitalocean (master|node) create --ssh-key` #1723 (changes to `~core/.ssh/authorized_keys` get reverted at boot)
* `kontena digitalocean (master|node) create --version` #2289 (agent reboots with the wrong version after master version upgrades)
* `kontena digitalocean master create --ssl-cert` #912 (need to use the non-cloudconfig'd `/etc/kontena-server.local.env` to update)
* `kontena digitalocean master create --mongodb-uri` (dunno about this one, it's written directly into the systemd unit, would need a drop-in to override)